### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by ztybigcat**
Updated README to a nicer format and linking to EC2 docker hosting guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-Create your *own branch* to do implemetation.
-After implementation, push the changes to your branch then create a pull request to dev branch.
-DO NOT PUSH TO MAIN!
+### **Why is Our Dataset Special? 🌟**
+
+1. **A Heart for Impaired Speakers**: Our platform stands out because it focuses on impaired speakers, making sure their voices are heard and recognized.
+
+2. **A Treasure Trove of Data**: We're proud to offer a dataset packed with high-quality audio from dysarthria speakers, detailing different severity levels. As of now, this is the most extensive public dataset in this unique domain.
+
+3. **Powered by Cutting-Edge Tech**: We've embraced wav2vec 2.0, a top-notch pre-trained speech recognition model, to refine our validation process. This ensures pristine data quality and also provides additional insights like intelligibility scores.
```
Comment (aanchan) [inline README.md]: I removed wav2vec2.0 on this @ztybigcat

Comment (aanchan) [reply README.md]: We do not do ASR validation on it, the models were too heavy.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
-# VoiceCollector
+## Welcome to Voice Collector! 🎙️
 
+Hello there! We're thrilled to have you here. Our application is not just any ordinary data collection platform. We're on a mission to make voice recognition inclusive for everyone, especially for those with speech impairments.
 
-## Dev Guide
-Create your *own branch* to do implemetation.
-After implementation, push the changes to your branch then create a pull request to dev branch.
-DO NOT PUSH TO MAIN!
+### **Why is Our Dataset Special? 🌟**
```
Comment (aanchan) [inline README.md]: We don't have a dataset yet :D

File: README.md (hunk 1 above)
Comment (aanchan) [reply README.md]: I would write a bit more about the layout of our project, sounds dry but what our front-end and backend are in and so on.

File: server/README.md
Comment (aanchan) [inline server/README.md:1]: We probably need to remove this README.md

File: web/README.md
Comment (aanchan) [inline web/README.md:1]: Like the server README.md, do we need this README.md here?

File: server/torgo_results.txt
Comment (aanchan) [inline server/torgo_results.txt:1]: This can be deleted.

File: server/tmp/normal.wav
Comment (aanchan) [inline server/tmp/normal.wav:1]: This I think can be deleted as well.

File: server/tests/test_convert_format.py
Comment (aanchan) [inline server/tests/test_convert_format.py:1]: Not sure if this test file is actually used

File: server/app.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Google App Engine config file
```
Comment (aanchan) [inline server/app.yaml]: Do we really need all these comments here?

File: server/app/scripts/read_audio.py
Comment (aanchan) [inline server/app/scripts/read_audio.py:1]: This file might actually not be used.

File: web/public/favicon.ico
Comment (aanchan) [inline web/public/favicon.ico:1]: We might need a new favicon

File: web/public/logo192.png
Comment (aanchan) [inline web/public/logo192.png:1]: Possibly logos as well, if these are not used, should be deleted.

File: web/public/logo512.png
Comment (aanchan) [inline web/public/logo512.png:1]: This also if not used should be deleted.

File: web/src/assets/1.jpg
Comment (aanchan) [inline web/src/assets/1.jpg:1]: This can be deleted

File: web/src/assets/2.jpg
Comment (aanchan) [inline web/src/assets/2.jpg:1]: This can also be deleted.

File: web/src/assets/3.jpg
Comment (aanchan) [inline web/src/assets/3.jpg:1]: delete

File: web/src/components/AboutUs.js
Comment (aanchan) [inline web/src/components/AboutUs.js:1]: This can certainly be deleted, we do not need an About Us section

File: web/src/components/ContactUs.js
Comment (aanchan) [inline web/src/components/ContactUs.js:1]: We need this to be deleted as well.

File: web/src/data.js
Comment (aanchan) [inline web/src/data.js:1]: I think this can be deleted.

File: web/src/image.js
Comment (aanchan) [inline web/src/image.js:1]: I think this is also not used, can be deleted

File: web/src/pages/Home.js
Comment (aanchan) [inline web/src/pages/Home.js:1]: This home page needs to be changed so that we can head directly to the record page

**Review by aanchan — CHANGES_REQUESTED** [review]

File: server/README.md
Comment (ztybigcat) [reply server/README.md:1]: Removed

File: web/README.md
Comment (ztybigcat) [reply web/README.md:1]: Removed

File: server/torgo_results.txt
Comment (ztybigcat) [reply server/torgo_results.txt:1]: Removed

File: server/tmp/normal.wav
Comment (ztybigcat) [reply server/tmp/normal.wav:1]: Removed

File: server/tests/test_convert_format.py
Comment (ztybigcat) [reply server/tests/test_convert_format.py:1]: Removed

File: server/app/scripts/read_audio.py
Comment (ztybigcat) [reply server/app/scripts/read_audio.py:1]: Removed

File: server/app.yaml (hunk 3 above)
Comment (ztybigcat) [reply server/app.yaml]: Removed relevant config files for google cloud.

File: web/public/favicon.ico
Comment (ztybigcat) [reply web/public/favicon.ico:1]: replaced with happy prime icons

File: web/public/logo192.png
Comment (ztybigcat) [reply web/public/logo192.png:1]: replaced with happy prime logos

File: web/public/logo512.png
Comment (ztybigcat) [reply web/public/logo512.png:1]: replaced with happy prime logos

File: web/src/assets/1.jpg
Comment (ztybigcat) [reply web/src/assets/1.jpg:1]: Removed

File: web/src/assets/2.jpg
Comment (ztybigcat) [reply web/src/assets/2.jpg:1]: Removed

File: web/src/assets/3.jpg
Comment (ztybigcat) [reply web/src/assets/3.jpg:1]: Removed

File: web/src/components/AboutUs.js
Comment (ztybigcat) [reply web/src/components/AboutUs.js:1]: Removed

File: web/src/components/ContactUs.js
Comment (ztybigcat) [reply web/src/components/ContactUs.js:1]: Removed

File: web/src/data.js
Comment (ztybigcat) [reply web/src/data.js:1]: Removed

File: web/src/image.js
Comment (ztybigcat) [reply web/src/image.js:1]: Removed

File: web/src/pages/Home.js
Comment (ztybigcat) [reply web/src/pages/Home.js:1]: Now user will directly land at record page.

File: README.md (hunk 2 above)
Comment (ztybigcat) [reply README.md]: Section removed

File: README.md (hunk 1 above)
Comment (ztybigcat) [reply README.md]: Replaced those content with an overview of different technologies used in the application.

**Pull request merged**